components/forms/AddToGroupsForm.js

```diff
@@ -8,7 +8,11 @@ export default function AddToGroupsForm({ groups }) {
     const { values, errors, touched } = useFormikContext();
     const { capabilities } = useContext(CacheContext);
 
-    const systemSupplied = groups.filter((group) => group.isSystemSupplied);
+    const systemSupplied = groups.filter((group) => {
+        if (group.id === SUPER_ADMIN_ID && !capabilities.includes(promoteToSuperAdmin)) return;
```

Comment:
We do not check group membership. We do not deal with groups. The concept of `groups` is an illusion. It does not exist. We only deal with capabilities.